src/get-element-override.js

```diff
@@ -1,11 +1,11 @@
 export default function getElementOverride (element, overrides) {
   const parentNode = getTopMostElementParent(element)
 
-  for (let override in overrides) {
+  for (const override in overrides) {
     if (overrides.hasOwnProperty(override)) {
       try {
-        const matches = parentNode.querySelectorAll(override)
-        if (Array.from(matches).indexOf(element) > -1) {
+        const matches = parentNode.getElementsByTagName(override)
```

Comment:
Considering that HTML2React supports all CSS selectors, `getElementsByTagName` will simply not cut it. The usage is documented in the [README.md](https://github.com/Deschtex/html2react/blob/master/README.md).